aerolyzer/image_restriction_main.py

```diff
@@ -0,0 +1,55 @@
+'''
+Image Restriction Main File
+Description: This file is called by the website when a photo is uploaded, and uses
+information from image_restriction_functions and image_restrictions_conf to verify that
+all image restrictions have been met.
+'''
+
+import sys
+
+from image_restriction_functions import imgRestFuncs as Functions
+
+'''
+Purpose:        The purpose of this function is to call each restriction check and print out the results
+Inputs:         dict exifData
+Outputs:        restriction check results
+Returns:        N/A
+Assumptions:    N/A
+'''
+def program(exifData, functions):
+    #Call each function and check for false return values
+    print functions.err_msg(functions.is_accepted_device(exifData), 'is_accepted_device')
```

Comment:
Here, each function doing a check should be called instead of this print string for error msgs. Check the tutorial on assertions.